code/modules/unit_tests/icon_state_sanity.dm

```diff
@@ -0,0 +1,15 @@
+/datum/unit_test/icon_state_sanity/Run()
+	var/list/all_objs = typesof(/obj)
+	var/list/all_turfs = typesof(/turf)
+	var/list/all_mobs = typesof(/mob)
+	var/list/atoms_to_check = all_mobs | all_objs | all_turfs
```

Comment:
`typesof(/atom) - typesof(/area)`? We want to check `/atom/movable` too, we have a few of those.

Reply:
yea, that does it better actually